locales/cy/translation.json

```diff
@@ -372,22 +372,22 @@
       }
     },
     "drivingLicenceDetailsNotCorrect": {
```

Comment:
I think need to regenerate snapshot tests for cy :)

Reply:
thanks! forgot about this